powerlibs/aws/process_status_notifier/__init__.py

```diff
@@ -13,6 +15,10 @@ def __init__(self, queue_name, process_name, topics_format='{process_name}__{sta
 
     @cached_property
     def notifier(self):
+        aws_region = os.environ.get('AWS_REGION_NAME')
+        if aws_region:
+            return SQSPublisher(aws_region=aws_region)
```

Comment:
Writing the call twice seems really awkward...
I suggest trying both `AWS_REGION_NAME` and `AWS_REGION`, since the last is used internally on the `SQSPublisher`, anyway.
Thus you'll be **alway** setting `aws_region` and calling `SQSPublisher` only once.